lib/iex/lib/iex.ex

```diff
@@ -222,6 +222,16 @@ defmodule IEx do
       ...(1)> #iex:break
       ** (TokenMissingError) iex:1: incomplete expression
 
+  ## Exiting the shell
+
+  There are a few ways to quit the IEx shell:
+
+    * via the `BREAK` menu (available via `Ctrl+C`) by typing `q`, `Enter`
+    * by hitting `Ctrl+C`, `Ctrl+C`
```

Comment:
Wikipedia uses `+` for `Ctrl+Alt+Del`: https://en.wikipedia.org/wiki/Control-Alt-Delete